doc/source/flexibility/maximum_binlog_size.rst

```diff
@@ -0,0 +1,57 @@
+.. _maximum_binlog_size:
+
+========================================
+ Restricting the number of binlog files
+========================================
+
+Estimating the number of binary logs and disk space use is a challenge. |Percona Server| has the variable :variable:`binlog_space_limit`, which sets the maximum amount of space to use for all binary logs. With the :variable:`binlog_space_limit` set to `0`, the default value, the variable is turned off and no limit is set. You set the :variable:`binlog_space_limit` on and provide a size limit. The value specified is not a hard limit. A single transaction is written to a single binary log, and the size of this transaction may exceed the limit.
+
+For example, the :variable:`max_binlog_size` is set to 1GB, which is the default value and the maximum value and the :variable:`binlog_space_limit` is set to 10GB. A large transaction changes 5GB of data. To store the transaction, the :variable:`binlog_space_limit` removes old binary logs as needed, up to the three oldest ones, to remain under the limit.
```

Comment:
"default value and the maximum value and the <variable>" - there are definitely some extra words here.

I also don't understand the example - why three? Why does it have to be one transaction? And it doesn't mention the current binlog size, only that the maximum is 10g - so if currently the sum of binlog size is 3 or 4g, this variable will do nothing.

Reply:
Removed most of the example text.